README.md

```diff
@@ -49,16 +49,22 @@ Finally, to run the project locally:
 npm run dev
 ```
 ### OpenApi documentation
-Open Api documentation is managed through [redoc](https://redocly.com/docs/redoc/). Changes to the documentation are managed through edits to the `openapi/openapi.yaml` file. The `openapi/index.html` is used to serve the rendered documentation. It is generated through the redocly cli and should not be manually edited.
-
-The api serves the documentation on its root path.
-
-- To serve a development version of the rendered documentation while making edits, use `npm run redoc:dev`.
+Open Api documentation is managed through [redoc](https://redocly.com/docs/redoc/). 
+Changes to the documentation are managed through edits to the `openapi/openapi.yaml` file. 
+The `openapi/index.html` is used to serve the rendered documentation. It is generated through the redocly cli and should not be manually edited.
+*It was historically a source-controlled file. However, we found it is more practicle to generate it as part of the API development and production build steps.*
```

Comment:
nit: practical*

Reply:
Is this aside really necessary? If someone comes across the checked-in `index.html` in the git history, they should be able to find this PR, which explains this context.